molecule/default/tasks/scale.yml

```diff
@@ -45,6 +45,37 @@
         field_selectors:
           - status.phase=Running
 
+    - name: Scale the deployment (check_mode)
```

Comment:
What about also trying to verify the idempotency overall?

Reply:
This is was not part of the change, I am just validating the check_mode
Idempotency should be addressed in another PR

Reply:
Generally in the aws collections we used to add tasks for idempotency even for check_mode.

Reply:
@gravesm What do you suggest?

Reply:
yes but in AWS collection you do not check the status of the resource, so checking the deployment after running the task using check_mode is another way to test the idempotency

Reply:
I don't think I get your point, but since the task in run in check_mode we don't expect to have any real change on the resource. With the idempotency in general, I mean running the same task twice.

Reply:
by the way it is done